docs/change-notes/migrating-from-v1-to-v2/federation-changes.md

```diff
@@ -242,21 +242,12 @@ By default, `GetSchema` results will be merged with `MergeSchema`. This returns
 
 
 
-## SuffixCacheName 
+## Cache Name Suffixes
 
-All Cache implementations are given a name and you can have multiple caches like NamedOperations, NamedViews, etc.
+Gaffer Caches now include suffixes in the names of cache entries. This allows for multiple cache entries for different graphs to co-exist using the same cache implementation without any conflicts.
 
-However within FederatedStore where multiple graphs may exist, a cache name has the suffix of `graphId` added by default for identity and keeping caches for different graphs separate, e.g NamedOperationCache_graph1, NamedOperationCache_graph2, NamedOperationCache_graph3.
+These suffixes can be customised which allows for graphs to use the same cache if desired, e.g. load balancing Federated Store instances.
```

Comment:
This would only work if they're in the same JVM. If you had some remotely accessible cache, then you might want separate cache instances that just point to a remote cache instead.

Reply:
Do you mean sharing a remote cache by having two different cache instances, where each pointed to the same remote cache? That would work, but what I meant by "cache" here is actually "cache entry". As in by using the same suffix two graphs would use the same cache entry (e.g. for named ops). This would apply equally to remote caches.

Maybe I should change this to read "cache entry" to similar.

Reply:
Yes I do mean that and I think it is more reliable than pointing to the same JCS cache for example. Perhaps cache entry or cache object? I am unsure

Reply:
I've clarified this with "cache entry".